modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManager.vue

```diff
@@ -203,6 +227,11 @@ export default {
     this.modal.active = true;
     this.getPieces();
   },
+  watch: {
+    checked: function() {
+      this.generateUi();
```

Comment:
Is this needed to disable them when the max is reached? The table mixin is also doing this when watching `rows`.

Reply:
Please add a `// NOTE:` comment to revisit this in the refactor.

Reply:
`rows` are not updated after every click to check/uncheck items. `checked` is. Done for the note.